.env

```diff
@@ -0,0 +1 @@
+MDBOOK_VERSION=0.4.4
```

Comment:
Keeps the version stable and makes sure we can update it more easily than changing the GH-Action all the time.

Reply:
for simplicity's sake I would prefer to write the version in `mdbook-version` directly. But if you prefer like this it's ok.

Reply:
yes, I find it much cleaner like this